composer.json

```diff
@@ -9,21 +9,25 @@
         }
     ],
     "require": {
-        "PHP": ">=7.0.0",
-        "symfony/http-foundation": "^3.0.9|^4.0.4"
+        "php": "^7.0",
+        "symfony/http-foundation": "^2.8 || ^3.0 || ^4.0"
     },
     "require-dev": {
-        "symfony/form": "^3.0.9|^4.0.4",
-        "phpunit/phpunit": "^6.5.6"
+        "symfony/form": "^2.8 || ^3.0 || ^4.0",
```

Comment:
Hi @hshn, do you have a reason why we would need to pin the versions to that fix part (`3.0.9` and `4.0.4`)? The way that I did should support fix and minor updates, which have a well managed BC promise by the Symfony project.

I will try to update travis.

Reply:
@andersonamuller 

The reason why I set version like `^4.0.4` is that, there's no necessary to support lower patch-version libraries. Because I think supporting lower patch-version libraries mean to support bugs which higher patch-version doesn't have. Besides that, it's very easy to upgrade patch-version of any Symfony libraries. But if both work well, there's no reason to drop lower version support as well.

Anyway, thank you for your work!